packages/backend/src/peer/model.ts

```diff
@@ -48,6 +51,27 @@ export class Peer
 
   public name?: string
 
+  public async onDebit({ balance }: OnDebitOptions): Promise<Peer> {
+    if (this.liquidityThreshold !== null) {
+      if (balance <= this.liquidityThreshold) {
+        await WebhookEvent.query().insert({
+          type: 'liquidity.peer',
+          data: {
+            id: this.id,
+            asset: {
+              id: this.asset.id,
+              code: this.asset.code,
+              scale: this.asset.scale
+            },
+            liquidityThreshold: this.liquidityThreshold,
+            balance
+          }
+        })
+      }
+    }
+    return this
+  }
+
```

Comment:
What is the thinking for putting this on the model? Perhaps this is more for my own understanding because I'm not that familiar with how we normally create webhooks, but I would have expected this to be a method on the asset service that calls a `webhookService.create` method.

Reply:
The reason I put it on the model rather than the service is to be able to check whether a liquidity account that is part of a transfer has that property and if so, I'll invoke the function. When we do a transfer, we don't distinguish between what kind of liquidity accounts are involved. We just deal with `sourceAccount` and `destinatioAccount`, without knowing the underlying model. Payments for instance don't have the `onDebit` property, but incoming payments have the `onCredit` property. It is currently invoked in the `post` of the transfer. 
https://github.com/interledger/rafiki/blob/7be0fa9b530bea4e2507b7d40ceafcfb9db0ed15/packages/backend/src/accounting/service.ts#L135-L154
My PR adds the `onDebit` part there too.

Reply:
Thanks for detailing that. Makes a bit more sense 😄